app/src/auto_validator/core/utils/ssh.py

```diff
@@ -26,22 +26,19 @@ def connect(self) -> bool:
                 passphrase=self.passphrase,
             )
         except Exception as e:
-            self.logger.exception(f"SSH Execute Command Error: {e}")
+            self.logger.exception("SSH Execute Command Error: %s", e)
```

Comment:
What command execution are we logging here about?

This is a connecftion attempt, not a command